mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -188,9 +206,11 @@ CLASS net/minecraft/unmapped/C_usxaxydn net/minecraft/entity/LivingEntity
 	METHOD m_epkzhkqg deserializeBrain (Lcom/mojang/serialization/Dynamic;)Lnet/minecraft/unmapped/C_rjqjaxef;
 		ARG 1 dynamic
 	METHOD m_exbsyzzv markEffectsDirty ()V
+	METHOD m_exhdcppw getSkipExperienceDrop ()Z
```

Comment:
What? What does this mean. Skip? Is this getting the skipped experience orb that was supposed to be dropped? Idk this sounds weird it may need a rename

Reply:
It does mean that, not sure how else to rename though

Reply:
yarn calls it `isExperienceDroppingDisabled`, i would say `shouldSkipDroppingExperience`

Reply:
Agree with oro